viz.yaml

```diff
@@ -0,0 +1,68 @@
+vizlab: "0.3.6"
+info:
+  id: gage-conditions
+  name: USGS Gage Conditions
+  date: 2018-06-11
+  publish-date: 2019-06-25
+  path: /gage-conditions
+  analytics-id: # WE WILL NEED TO FILL THIS IN AT SOME POINT
+  description: Map of all USGS gages and their conditions.
+  required-packages:
+    vizlab:
+      repo: github
+      version: 0.3.6
+      name: USGS-VIZLAB/vizlab
+      ref: v0.3.6
+    dataRetrieval:
+      repo: CRAN
+      version: 2.7.4
+    dplyr:
+      repo: CRAN
+      version: 0.7.4
+  contributors:
+    -
+      name: Lindsay Carr
+      email: lcarr@usgs.gov
+      affiliation: U.S. Geological Survey
+    -
+      name: Jordan Read
+      email: jread@usgs.gov
+      affiliation: U.S. Geological Survey
+parameter:
+  - id: plot_date
+    date: "2018-05-01"
+fetch:
+  - id: dv_sites
+    location: cache/dv_sites.json
+    reader: json
+    fetcher: dv_sites
+    scripts: [scripts/fetch/dv_sites.R]
+    depends:
+      date: plot_date
+  - id: site_stats # took about 20 min
+    location: cache/site_stats.json
+    reader: json
+    fetcher: site_stats
+    scripts: [scripts/fetch/site_stats.R]
+    depends:
+      sites: dv_sites
+  - id: dv_data # took about 30 min
```

Comment:
It would be more data, but curious how much longer this fetch would take if we had it pull the whole `year` for the site instead of just the day? That would be a (hopefully) slightly longer initial pull, but would give us more flexibility in changing the plot date w/o an expensive rebuild and also (like above comment) would like this code up to more easily accept a multi-day viz pattern.

Reply:
Took 45 minutes compared to 30, so probably extra time well spent.